src/main/java/uk/gov/hmcts/probate/service/filebuilder/IronMountainFileService.java

```diff
@@ -171,6 +171,10 @@ private String getCaseTypeMapped() {
         }
     }
 
+    private String registryLocationCheck(String registry) {
+        return registry.equalsIgnoreCase("ctsc") ? "Principal Registry" : registry;
```

Comment:
Can we put some of these String values as statics please?
And your CaseTypeMapping enum on this class should be renamed as a InronMountain specific (possibly in its own class)

Reply:
👍